Library/SubscriptionChange.cs

```diff
@@ -110,14 +108,17 @@ internal void WriteChangeSubscriptionXml(XmlTextWriter xmlWriter)
             if (CollectionMethod.Like("manual"))
             {
                 xmlWriter.WriteElementString("collection_method", "manual");
-                xmlWriter.WriteElementString("net_terms", NetTerms.Value.AsString());
-                xmlWriter.WriteElementString("po_number", PoNumber);
             }
             else if (CollectionMethod.Like("automatic"))
             {
                 xmlWriter.WriteElementString("collection_method", "automatic");
             }
 
+            if (NetTerms.HasValue)
+                xmlWriter.WriteElementString("net_terms", NetTerms.Value.AsString());
+
+            xmlWriter.WriteStringIfValid("po_number", PoNumber);
```

Comment:
Even though the API requires a manual invoice for these values, the client shouldn't be checking to see if the CollectionMethod is manual.